mteb/tasks/Clustering/fra/HALClusteringS2S.py

```diff
@@ -46,3 +50,64 @@ def dataset_transform(self):
             "labels": [split.tolist() for split in np.array_split(domains, 10)],
         }
         self.dataset["test"] = datasets.Dataset.from_dict(new_format)
+
+
+class HALClusteringS2SFast(AbsTaskClusteringFast):
+    metadata = TaskMetadata(
+        name="HALClusteringS2SFast",
+        description="Clustering of titles from HAL (https://huggingface.co/datasets/lyon-nlp/clustering-hal-s2s)",
+        reference="https://huggingface.co/datasets/lyon-nlp/clustering-hal-s2s",
+        dataset={
+            "path": "lyon-nlp/clustering-hal-s2s",
+            "revision": "e06ebbbb123f8144bef1a5d18796f3dec9ae2915",
+        },
+        type="Clustering",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["fra-Latn"],
+        main_score="v_measure",
+        date=("2000-03-29", "2024-05-24"),
+        form=["written"],
+        domains=["Academic"],
+        task_subtypes=["Thematic clustering"],
+        license="Apache-2.0",
+        socioeconomic_status="medium",
+        annotations_creators="derived",
```

Comment:
Thank you for filling this!
HAL is like Arxiv for French academics, I think annotations are created by authors of the articles when submitting, can we say it's `human-annotated` then?